package.json

```
@@ -5,37 +5,24 @@
   "author": "NRK <opensource@nrk.no> (https://www.nrk.no/)",
   "version": "4.1.0",
   "license": "MIT",
-  "main": "index.js",
+  "main": "core-icons.cjs",
```

see above about renaming
```suggestion
  "main": "core-icons.js",
```